photosynthesis_metrics/gmsd.py

```diff
@@ -0,0 +1,253 @@
+"""
```

Comment:
Only classes were defined here. Should we define the methods for GMSD and MS-GMSD as we did for SSIM and MS-SSIM?

Reply:
Why? The reason to have separate class for `SSIM` is that when it's used as a measure it's value _maximised_ and when used as a loss it's value _minimised_. Here for both cases value of GMSD is minimised, so class `GMSDLoss` can be used as a measure without any additional changes.

Reply:
Well, actually no. As I understood, the solution was to return true value and describe it in the documentation that user should take it into account.